apiary.apib

```diff
@@ -3281,6 +3281,327 @@ Update the current timer. Only possible if there is a timer running.
 
 + Response 204
 
+# Group Work Orders
+
+## Work Orders [/workOrders]
+
+### workOrders.draft [POST /workOrders.draft]
+
+Add a draft work order.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + customer (object, required)
+            + type (enum[string], required)
+                + Members
+                    + contact
+                    + company
+            + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
+        + starts_at: `2017-05-26T10:01:49+00:00` (string, required)
+        + description: `Pipework intervention` (string, optional)
+        + department_id: `cef01135-7e51-4f6f-a6eb-6e5e5a885ac7` (string, optional)
+        + layout_id: `a05320f6-b3ef-4ff9-a287-2074a12fb3e9` (string, optional)
+        + milestone_id: `e94c1363-3426-46a4-921d-fce03f39bbd5` (string, optional)
+        + event_id: `9fc14045-5c6c-4ba8-8672-42ea3f26aa63` (string, optional)
+        + location (Address, optional)
+        + distance (object, optional)
```

Comment:
Q: Should we define the `distance` object in `datastructures.apib`?

Reply:
Extracted it. There might be more possible data structures though.

Reply:
Would it make sense to add `duration` to the `datastructures.apib`? I considered it but went with the safer option (not make it global) but since we're adding it here too, might make sense to make it a shared data structure.